gradle.properties

```diff
@@ -14,7 +14,7 @@ pluginUntilBuild=
 
 # IntelliJ Platform Properties -> https://github.com/JetBrains/gradle-intellij-plugin#intellij-platform-properties
 platformType = IC
-platformVersion = 2020.3
+platformVersion = 2022.2
```

Comment:
I'm not so sure we should do this... 🤔 

as long as we're supporting 2020.3 as our minimum version, I think we should strive to keep compatibility of our tests to that version. 

Let's **not** merge this change.

Reply:
There are tradeoffs.
The Plugin Verifier workflow that executes on each pr actually runs our tests for each supported IDE version (from our minimum up to the latest), except from the UI tests.
So, this change is essentially about the IDE version used by the UI tests and the version used when developing/testing the plugin locally.
Given that UI tests are unstable, I think it's a good compromise:
+ Plugin compatibility with older versions is still ensured/tested
+ UI tests become stable (?) // still assuming we overcome the last issue wrt clicking IDE buttons

Reply:
I am not yet convinced that changing this version is what fixes the tests. 

Having said that, I also don't want to block progress, so: 
* I'll resolve this 
* we merge this PR 
* I'll open another PR that reverts this change so we can see if that still works. If not, we keep it.